Taylor_series_Repo/Taylor_series_Horner_rule.c

```diff
@@ -0,0 +1,59 @@
+/*Description of Algorithm:-

```

Comment:
Why Taylor_series_Horner_rule.c is there?

Reply:
I don't why it is there. This will happen after I squash the commits. Please Tell me why this happens and how to resolve it. Thank you @GOVINDDIXIT